walking_skeleton/laser_comms/laser_comms/test.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
@khorassa please delete the walking_skeleton folder, because we don't use it any more

Reply:
yep ok